packages/core/src/components/date-picker-legacy/readme.md

```diff
@@ -30,7 +30,7 @@ Based on the [Duet Date Picker](https://duetds.github.io/date-picker-legacy/).
 | Event           | Description                                         | Type                                                                                                                                                                               |
 | --------------- | --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `dsoBlur`       | Event emitted the date picker input is blurred.     | `CustomEvent<{ component: "dso-date-picker-legacy"; }>`                                                                                                                            |
-| `dsoDateChange` | Event emitted when a date is selected.              | `CustomEvent<{ component: "dso-date-picker-legacy"; valueAsDate: Date \| undefined; value: string; error?: "required" \| "min-range" \| "max-range" \| "invalid" \| undefined; }>` |
+| `dsoDateChange` | Event emitted when a date is selected.              | `CustomEvent<{ component: "dso-date-picker-legacy"; valueAsDate: Date \| undefined; value: string; error?: "invalid" \| "required" \| "min-range" \| "max-range" \| undefined; }>` |
```

Comment:
Deze wijziging hebben we even gemist, want lokaal komt ie nu elke keer naar voren...

Reply:
Ja, gevonden! Ik ga 'm terugzetten. Deze change wordt in een nieuwe commit op master toegepast

Reply:
Zie 133cd652dc8405b938f73ad2ad93c8895e870fad

Reply:
Ah, check, je hebt mijn opmerking gevonden. 👍